tests/flytekit/unit/models/test_concurrency.py

```diff
@@ -0,0 +1,41 @@
+from flyteidl.admin import launch_plan_pb2 as _launch_plan_idl
+
+from flytekit.models.concurrency import ConcurrencyLimitBehavior, ConcurrencyPolicy
+
+
+def test_concurrency_limit_behavior():
+    assert ConcurrencyLimitBehavior.SKIP == _launch_plan_idl.CONCURRENCY_LIMIT_BEHAVIOR_SKIP
+
+    # Test enum to string conversion
+    assert ConcurrencyLimitBehavior.enum_to_string(ConcurrencyLimitBehavior.SKIP) == "SKIP"
+    assert ConcurrencyLimitBehavior.enum_to_string(999) == "<UNKNOWN>"
+
+
+def test_concurrency_policy():
```

Comment:
```suggestion
def test_concurrency_policy_serialization():
```
nit: making test name more clear